src/ui/utils/visualizations/line.ts

```diff
@@ -23,7 +24,11 @@ export function linearCheckColor({
     designItems,
     isMultipleColorsSupported,
 }: CommonCheckColorArgs) {
-    if (isMeasureField(item) || isMeasureValue(item)) {
+    if (
+        isMeasureField(item) ||
+        isMeasureValue(item) ||
+        item.data_type === DATASET_FIELD_TYPES.MARKUP
```

Comment:
Is this check really necessary here? Aren't the available field types checked elsewhere?

Reply:
No, this exact function checks available fields for colors section